data_pipelines_cli/config_generation.py

```diff
@@ -0,0 +1,29 @@
+import pathlib
+
+import yaml
+from dbt_airflow_manifest_parser.config_utils import read_config
+
+from .cli_constants import BUILD_DIR, profiles_build_path
+
+
+def generate_profiles_yml(env: str) -> pathlib.Path:
+    dbt_env_config = read_config(BUILD_DIR.joinpath("dag"), env, "dbt.yml")
+    dbt_target = dbt_env_config["target"]
+    dbt_target_type = dbt_env_config["target_type"]
+
+    target_type_config = read_config(
+        BUILD_DIR.joinpath("dag"), env, f"{dbt_target_type}.yml"
+    )
+
+    profile = {
+        dbt_target_type: {
+            "target": dbt_target,
+            "outputs": {dbt_target: target_type_config},
+        }
+    }
```

Comment:
I would extract it to function called: generate_profile_json

Reply:
extracted